hedera-node/src/test/java/com/hedera/services/state/exports/SignedStateBalancesExporterTest.java

```diff
@@ -231,10 +231,8 @@ public void logsOnSigningFailure() {
 	public void usesNewFormatWhenExportingTokenBalances() throws IOException {
 		// setup:
 		ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
-		Pattern CSV_NAME_PATTERN = Pattern.compile(
-				".*\\d{4}-\\d{2}-\\d{2}T\\d{2}_\\d{2}_\\d{2}[.]\\d+Z_Balances.csv");
```

Comment:
I think it is actually better to expect this pattern in our test, to make sure that `DateTimeFormatter.ISO_INSTANT.format(when).replace(":", "_") + "_Balances.csv"` really yields it. Maybe make it stricter by expecting `d{9}` before `Z` instead of `d+`

Reply:
It doesn't always produce d{9}. Sometimes it produces 1 digit for nanos. There's a ticket requesting it to always be 9 but it's not scheduled.

Reply:
@steven-sheehy, could you please provide the ticket/issue?

Reply:
https://github.com/hashgraph/hedera-services/issues/483

Reply:
@steven-sheehy, this is for record stream. Will fix that as well! Thanks!